financial_assistant/src/llm.py

```diff
@@ -8,7 +8,7 @@
 from langchain_core.output_parsers import JsonOutputParser
 from langchain_core.prompts import PromptTemplate
 from langchain_core.tools import StructuredTool, Tool
-from pydantic import BaseModel
```

Comment:
consider changing this  for `from pydantic import BaseModel`
 keeping langchain pydantic_v1 will create conflict with LC0.3.1

Reply:
I will update all the `pydantic` dependencies when I upgrade my package to `langchain` 3.0.
Thank you for the reminder.

Reply:
I will open a PR with this upgrade soon.

Reply:
Now reverted to `pydantic`. Thank you!